guides/common/modules/proc_creating-compute-profiles.adoc

```diff
@@ -8,6 +8,12 @@ A default installation of {ProjectName} contains three predefined profiles:
 * `2-Medium`
 * `3-Large`
 
+You can apply the compute profiles to the following compute resource providers:
+
+* VMware
+* Libvirt
+* Red Hat Virtualization (deprecated)
+
```

Comment:
IMHO this list is missing Proxmox. cc @Manisha15

Reply:
Yes, proxmox should also be in the list.